docs/Actions/revoke-user-role.md

```diff
@@ -0,0 +1,197 @@
+                    "OnError": []
+                },
+                "$_uid": "action15922109647737176",
+                "$_isOpen": false,
+                "$_isLoaded": true,
+                "$_isFocus": true
+            },
+            {
+                "Title": "Execute actions on entity list",
+                "ActionType": "ExecuteActionsOnEntityList",
+                "Parameters": {
+                    "EntityName": "UserDetails",
+                    "Filters": "",
+                    "ContextBehavior": {
+                        "Expression": "",
+                        "Value": "",
+                        "IsExpression": false,
+                        "Parameters": {}
+                    },
+                    "ActionList": [
+                        {
+                            "Id": -1,
+                            "$_uid": "action15922109648872432",
+                            "Parameters": {
+                                "Id": "[UserDetails:UserId]",
+                                "Portal": ""
+                            },
+                            "ActionType": "LoadUser",
+                            "$_isOpen": false,
+                            "$_isLoaded": true,
+                            "$_isFocus": true,
+                            "Definition": {
+                                "IsClientAction": false,
+                                "Settings": {
+                                    "Group": "Context"
+                                }
+                            }
+                        },
+                        {
+                            "Id": -1,
+                            "$_uid": "action15922109648872488",
+                            "Parameters": {
+                                "RoleId": {
+                                    "Expression": "",
+                                    "Value": "7",
+                                    "IsExpression": false,
+                                    "Parameters": {}
+                                },
+                                "RoleNames": ""
+                            },
+                            "ActionType": "RevokeUserRole",
+                            "$_isOpen": false,
+                            "$_isLoaded": true,
+                            "$_isFocus": true,
+                            "Definition": {
+                                "IsClientAction": false,
+                                "Settings": {
+                                    "Group": "User"
+                                }
+                            }
+                        }
+                    ],
+                    "ContinueOnError": "",
+                    "OnError": []
+                },
+                "$_uid": "action15922109647731312",
+                "$_isOpen": false,
+                "$_isLoaded": true,
+                "$_isFocus": true
+            }
+        ]
+    }
+}
+```
+
+## `Frequently asked questions`
```

Comment:
Same as with Grant Role, remove this section and make sure the content falls naturally where it belongs.

Reply:
Done